app/src/main/AndroidManifest.xml

```diff
@@ -115,6 +126,10 @@
             android:name="com.example.musicdao.MusicGossipingService"
             android:enabled="true"
             android:exported="false" />
+
+        <meta-data
+            android:name="com.google.android.geo.API_KEY"
+            android:value="AIzaSyA3FH8vlkY_DyENUT5J7R48a8IRY9FUkJQ"/>
```

Comment:
I don't think you should have pushed this.

Reply:
I was wondering this. Since a GoogleMap is drawn within the app, I deemed it necessary. How to include it without showing it's value?

Reply:
Normally you'd put this in some config file and share it using some other method, but I suppose that's not too handy in an open-source project. But you've already pushed it so it's in the git history; let's keep it this way for now.

Reply:
It’s a key that I can remove easily so than it’s useless. I’ll look online to see what solutions there are.